morph/react-native/block-scroll-view.js

```diff
@@ -0,0 +1,16 @@
+import wrap from '../react/wrap.js'
+import { getProp, getVariableName, isScrollView } from '../utils'
+
+export function enter(node, parent, state) {
+  if (!isScrollView(node)) return
+
+  let scrollToEnd = getProp(node, 'scrollToEnd')
+  if (scrollToEnd) {
+    let variableName = getVariableName('scrollViewRef', state)
+    state.variables.push(`let ${variableName} = React.useRef()`)
+    state.render.push(` ref=${wrap(variableName)}`)
+    state.render.push(
+      ` onContentSizeChange={() => ${variableName}.current.scrollToEnd({ animated: true })}`
```

Comment:
would this make the container always scroll whenever a new item is added?

Reply:
Yes, I think that would be the effect. Whenever a new message is received it will scroll to the end, as otherwise it is always stuck at the place where it was previously set (e.g. even at the top of the conversation if the user didn't scroll)